### PR TITLE
Feature/us3843 request childcare

### PR DESCRIPTION
**Pull request by silbermm**

### CODE REVIEW

File: CI/SQL/20160519130900_US3843-add-childcare-locations-pageview.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+           ,[Page_ID]
+           ,[Field_List]
+           ,[View_Clause]
+           ,[Order_By])
+     VALUES
+           (990
+		   ,'Congregations With Childcare'
+           ,288
+           ,'Congregations.[Congregation_ID] AS [Congregation ID] , Congregations.[Congregation_Name] AS [Congregation Name] '
+           ,'Childcare_Contact_Table.[Contact_ID] IS NOT NULL '
```
Comment (IngageRitterS) [inline CI/SQL/20160519130900_US3843-add-childcare-locations-pageview.sql]: Spaces in the names - generally speaking if we are going to declare an alias using AS, there are underscores in the names, not spaces.  Seeing the names with spaces in the code makes me sad :(

File: crossroads.net/app/mp_tools/request_childcare/requestChildcare.controller.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    }
+  }
+
+  showGroups() {
+    return this.choosenCongregation && this.choosenMinistry && this.groups.length > 0 ;
+  }
+
+  formatPreferredTime(time) {
+    const startTimeArr = time['Childcare Start Time'].split(':');
+    const endTimeArr = time['Childcare End Time'].split(':');
```
Comment (IngageRitterS) [inline crossroads.net/app/mp_tools/request_childcare/requestChildcare.controller.js]: 'Childcare Start Time' - an example of spaces rather than underscores :(

File: CI/SQL/20160519130900_US3843-add-childcare-locations-pageview.sql (hunk 1 above)
Comment (plachmann) [inline CI/SQL/20160519130900_US3843-add-childcare-locations-pageview.sql]: Cheer up! The spaces are now gone. :)

File: crossroads.net/app/routes.js [hunk 3]
```diff
@@ -508,149 +508,8 @@
...
-          templateUrl: 'gp_export/gpExport.html'
-        })
-        .state('thedaily', {
-          parent: 'noSideBar',
-          url: '/thedaily',
-          controller: 'TheDailyController as dailyCtrl',
-          templateUrl: 'thedaily/thedaily.html'
         });
-
-    //Leave the comment below.  Once we have a true 404 page hosted in the same domain, this is how we
```
Comment (danrye) [inline crossroads.net/app/routes.js]: Why was this deleted?

File: crossroads.net/app/routes.js [hunk 4]
```diff
@@ -508,149 +508,8 @@
...
-              description: ''
-            }
-          }
-        })
-        .state('tools.gpExport', {
-          url: '/gpExport',
-          controller: 'GPExportController as gpExport',
-          templateUrl: 'gp_export/gpExport.html'
-        })
-        .state('thedaily', {
```
Comment (danrye) [inline crossroads.net/app/routes.js]: This needs added back

File: crossroads.net/app/routes.js [hunk 5]
```diff
@@ -508,149 +508,8 @@
               }).$promise;
             }
           }
-        })
-
-      // Commented out for US2924, will be added back after Corkboard go-live
```
Comment (danrye) [inline crossroads.net/app/routes.js]: This needs added back
